.github/workflows/periodic.yml

```diff
@@ -0,0 +1,67 @@
+name: periodic
+
+on:
```

Comment:
You might want to add

```
on:
  pull_request:
```

here just to test the workflow once, then remove the `pull_request` trigger after before landing.  `ciflow/periodic` won't work on this PR till it's land (chicken and egg issue)

Reply:
Triggering it one-off